docs/connect-blockchain/other/bitcoin.mdx

```diff
@@ -0,0 +1,220 @@
+---
+title: Integrate Web3Auth with the Bitcoin Blockchain
+sidebar_label: Bitcoin
+image: "guides/banners/bitcoin.png"
+displayed_sidebar: resources
+keywords: [bitcoin, taproot, web3auth, authentication, blockchain]
+description: "Integrate Web3Auth with the Bitcoin Blockchain | Documentation - Web3Auth"
+---
+
+import GetUserInfoSnippet from "@site/src/common/docs/_get-userinfo.mdx";
+import InitializeWeb3Auth from "@site/src/common/docs/_initialize-web3auth-other-pnp-sfa-tkey.mdx";
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+When using the Web3Auth Web SDK for non-EVM chains like [Bitcoin](https://developer.bitcoin.org/),
+you get a standard provider that allows you to access the user's private key. With this private key,
+you can use the appropriate Bitcoin blockchain libraries to perform various actions such as
+retrieving the user's account information, fetching their balance, signing transactions, and sending
+transactions. We've highlighted a few methods here to help you get started quickly.
+
+`Note: This documentation is for the Bitcoin Taproot integration.`
+
+## Installation
+
+```bash npm2yarn
+npm install --save bitcoinjs-lib ecpair axios
+```
+
+## Initializing Provider
+
+### Getting the `chainConfig`
+
+<Tabs
+ defaultValue="mainnet"
+  values={[
+    { label: "Mainnet", value: "mainnet", },
+    { label: "Testnet", value: "testnet", },
+  ]}
+>
+<TabItem
+  value="mainnet"
+>
+
+```typescript
+const chainConfig = {
+  chainNamespace: "other",
```

Comment:
use the state object, `CHAIN_NAMESPACES.OTHER` - I think this change should be there across all our chain config objects, latest chain chain creates issues if the object is not passed